tpl/bootstrap.php

```diff
@@ -66,7 +66,25 @@
 		<meta name="theme-color" content="#ffe57e" />
 	</head>
 	<body role="document">
-		<nav class="navbar navbar-default navbar-static-top">
+        <!-- Modal decryption password request -->
+        <div class="modal fade" id="passwordModal" role="dialog">
+          <div class="modal-dialog">
+            <!-- Modal content-->
+            <div class="modal-content">
+              <div class="modal-body">
+                <form role="form" id="passwordForm">
+                  <div class="form-group">
+                    <label for="decryptPassword"><span class="glyphicon glyphicon-eye-open"></span> <?php echo I18n::_('Please enter the password for this paste:')?></label>
+                    <input type="password" class="form-control" id="decryptPassword" placeholder="Enter password" autofocus>
+                  </div>
+                  <button type="submit" class="btn btn-success btn-block"><span class="glyphicon glyphicon-off"></span>Decrypt</button>
```

Comment:
Please localize it (`<?php echo I18n[…]`) and add a space before the word, as it currently looks too "compressed" otherwise:
![comp](https://cloud.githubusercontent.com/assets/11966684/20242530/541daa9e-a931-11e6-8b6d-9dd585a3c594.png)